src/types/jsx.d.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import 'react'
+
+declare module 'react' {
+  namespace JSX {
+    interface IntrinsicElements {
+      'va-accordion': any
```

Comment:
Ideally, we don't leave these as `any`, but I don't think they had proper types anyhow. 🤷 It'd be nice if we could import the prop (attribute) types for each web component, but I don't think the component library exports that. We don't want to use the prop types for the React bindings.